base32_lib/base32.py

```diff
@@ -42,19 +40,20 @@ def encode(number, split_every=0, min_length=0, checksum=False):
     :returns: A random Douglas Crockford base32 encoded string composed only
               of valid URI characters.
     """
-    assert isinstance(number, six.integer_types)
+    assert isinstance(number, int)
 
     if number < 0:
         raise ValueError("Invalid 'number'. Must be >= 0.")
 
     if split_every < 0:
         raise ValueError("Invalid 'split_every'. Must be >= 0.")
 
-    encoded = ''
     original_number = number
+
     if number == 0:
-        encoded = '0'
+        encoded = "0"
```

Comment:
i assume you changed to black formatting? should pytest-black been added to the test run?

Reply:
agree